source/meta.tex

```diff
@@ -1824,7 +1826,7 @@
  \tcode{struct is_nothrow_convertible;}     &
  \tcode{is_convertible_v<From, To>} is \tcode{true} and
  the conversion, as defined by \tcode{is_convertible},
- is known not to throw any exceptions\iref{expr.unary.noexcept} &
+ is not a potentially-throwing expression\iref{except.spec}. &
```

Comment:
We probably have another problem here, because a conversion is not an expression.